### ISSUE TITLE
system: do not drop DB when cleaning

### ISSUE THREAD

**Issue by tiborsimko** [cernopendata/opendata.cern.ch#2027]
The clean-instance.sh helper script currently destroys the DB, which is not good in DBoD usage scenarios. It should only empty the DB, not destroy it.

Let's rather have a clean-distinction strategy going like: 

- **provision X** - install whatever is necessary for a service X to run on the given platform; 
- **create X** - create instance or any necessary runtime services (COD3 instance, DB database, ES indexes, whatever)
- **start X** - start instance or service
- **populate instance** - upload records to the instance
- **clean instance** - "unpopulate" records from the instance, i.e. empty site content and its tables, but keep them otherwise ready; the site should still show, but be empty
- **destroy X** - delete COD3 instance and/or any necessary DB, ES resources

**Comment by ioannistsanaktsidis** [comment]
@hjhsalo do you want to take care of this one ?

**Comment by tiborsimko** [comment]
@ioannistsanaktsidis Yes, we'll take care of this on our end... I'll add a few more system tickets of this kind.
